tensorflow/python/eager/backprop.py

```diff
@@ -212,7 +212,7 @@ def grad_fn(*args, **kwds):
     if not variables:
       raise ValueError("No trainable variables were accessed while the "
                        "function was being computed.")
-
+    
```

Comment:
You have trailing whitespace now, line should just be empty.

Better yet, you can just try reverting this file to the original version in the repo.

Reply:
Now its perfectly fine with the original repo, i have checked.

Reply:
It is not, there's trailing whitespace and that will cause linters to error and not accept the PR.

This file should not show up in the "files changed" by this PR

Reply:
You can try running the following commands (make sure to copy them exactly, one by one):

```
git stash
git checkout master
git pull
git checkout -
git checkout master -- tensorflow/python/eager/backprop.py
git commit -m "Fixed whitespace"
git stash pop
git push
```